tests/expectations/parser/identifiers/reserved_identifier_fail.out

```diff
@@ -0,0 +1,5 @@
+Error [EPAR0370005]: expected ( -- found 'constructor'
```

Comment:
I guess the error message isn't ideal. When the parser doesn't find an identifier, it assumes it should find a list of identifiers in parens which is why we get this.